GoogleForCreators/web-stories-wp
# Added duration to video trimmer component

## Context

This adds duration below the video trimmer rail.

## Relevant Technical Choices

* Formats the duration in the same way video duration is formatted in the media panel.

## User-facing changes

![trim-duration](https://user-images.githubusercontent.com/637548/139600037-0242ed13-d8fe-4721-bd51-08a8b3d6306d.gif)

## Testing Instructions

This PR can be tested by following these steps:

1. Add a video
2. Enter trim mode
3. Adjust trimming handles
4. Observe the resulting video duration displayed below the trim rail

## Checklist




---

Fixes #9193
